### PR TITLE
Fix/#238 replace message emoji

### LINKED ISSUE #238: Replace fuselage's MessageEmoji component with our own component
1. Replace the fuselage's MessageEmoji component with our own component
2. The component should implement our theming system

### PR DESCRIPTION
**Pull request by umangutkarsh**
# Brief Title: Replace MesasgeEmoji

## Acceptance Criteria fulfillment

- [X] Task 1: Replace MessageEmoji component with own component

Fixes #238

### CODE REVIEW

**Comment by umangutkarsh** [comment]
Hey @abhinavkrin , please review the PR, and LMN if any changes are required or if there's any issue.
Thanks

**Review by abhinavkrin — COMMENTED** [review]
1. You should end the contents of a file with a new line. 
2. The MessageEmoji.js file is black.

File: .editorconfig [hunk 1]
```diff
@@ -7,4 +7,4 @@ insert_final_newline = true
 [*.{js,json,yml}]
 charset = utf-8
 indent_style = space
-indent_size = 2
+indent_size = 2
```
[the remark this answers is not in the document]
Comment (umangutkarsh) [reply .editorconfig]: Added new line in this file. Which other files should I edit?

[the remark this answers is not in the document]
Comment (abhinavkrin) [reply .editorconfig]: I would suggest checking the changed files of this PR.
Also, could you check why MessageEmoji.js is an empty file.

**Review by abhinavkrin — CHANGES_REQUESTED** [review]
1. End files with a new line. Check PR file changes to see which files are not ended with a LN. (recommended)
2. Please use an eslint plugin to check the styling errors and fix them (recommended)
3. Changes break the size of the emoji in the message and reaction list. It appears to be larger than it should be. (MUST) 
emoji-size-change

**Comment by umangutkarsh** [comment]
> 1. End files with a new line. Check PR file changes to see which files are not ended with a LN. (recommended)
> 2. Please use an eslint plugin to check the styling errors and fix them (recommended)
> 3. Changes break the size of the emoji in the message and reaction list. It appears to be larger than it should be. (MUST) 
> emoji-size-change
> 

@abhinavkrin, will soon look into it.

File: .gitignore [hunk 2]
```diff
@@ -10,3 +10,7 @@ node_modules
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+
+.babelrc
+
+.vscode
```
Comment (abhinavkrin) [inline .gitignore:16]: end with new line

File: yarn.lock [hunk 3]
```diff
@@ -33907,3 +33907,4 @@ proxy-middleware@latest:
   checksum: 80acd0fbf633782996642802c8692bbb80ae5c80a8dff4c501b88250acd5ccd468fbc6398bdce198475a25e3839c91385b81da921274f33ffb5c2d08c3eab400
   languageName: node
   linkType: hard
+
```
Comment (abhinavkrin) [inline yarn.lock:33910]: end with new line

**Review by abhinavkrin — CHANGES_REQUESTED** [review]
Almost read to merge.
1. Please commit only those files which are related to your changes
2. End files with new lines https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

**Comment by umangutkarsh** [comment]
> Almost read to merge.
> 
> 1. Please commit only those files which are related to your changes
> 2. End files with new lines https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

@abhinavkrin, I had to commit files other than those related to changes since earlier I was getting this error -> `no babel config file detected`. To resolve this, had to add some changes to `.eslintrc.json` file. But then I reverted those back since adding some configuration in `settings.json` resolved this error. 
But since I use `Prettier-Code formatter`, these code formatting are automatically applied when I saved the above files.

**Review by abhinavkrin — COMMENTED** [review]
Hey! Why is `.yarn/releases/yarn-berry.cjs` included in the commit and is an empty file?

**Comment by umangutkarsh** [comment]
> Hey! Why is `.yarn/releases/yarn-berry.cjs` included in the commit and is an empty file?

berry
This is `yarn-berry.cjs` on my local machine^

**Review by abhinavkrin — APPROVED** [review]
LGTM!

**Pull request merged**